// Copyright (c) 2011 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for an app setting.
// This is the same as for an extension setting, but uses a separate datatype
// in order to control syncability separately.

// Update proto_value_conversions{.h,.cc,_unittest.cc} if you change
// any fields in this file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;
option retain_unknown_fields = true;

package sync_pb;

import "sync.proto";
import "extension_setting_specifics.proto";

// Properties of app setting sync objects; just an extension setting.
message AppSettingSpecifics {
  optional ExtensionSettingSpecifics extension_setting = 1;
}

extend EntitySpecifics {
  optional AppSettingSpecifics app_setting = 103656;
}
